src/gsy_e/models/area/scm_manager.py

```diff
@@ -165,11 +167,22 @@ class CommunityData:
     energy_need_kWh: float = 0.
     energy_bought_from_community_kWh: float = 0.
     energy_sold_to_grid_kWh: float = 0.
+    trades: List[Trade] = None
```

Comment:
Did you use `__post_init__` to avoid issues with the list mutable type? If so, dataclasses have a specific approach for it:
```suggestion
    trades: List[Trade] = field(default_factory=list)
```

- [Docs: Mutable default values](https://docs.python.org/3/library/dataclasses.html#mutable-default-values)

Reply:
Let me try this, thanks!